src/index.ts

```diff
@@ -1,25 +1,24 @@
 let cantidad: number = 270;
 let mayores: number = 0;
-let menores: number = 0; 
+let menores: number = 0;
 let personasEnDisco: number[] = new Array(cantidad);
-let indice: number;
 
 function Aleatorio(menorValor: number, mayorValor: number): number {
-  return Math.floor(Math.random() * mayorValor || menorValor);
+  return Math.floor(Math.random() * (mayorValor + menorValor) + menorValor);
 }
 
-for (indice = 0; indice <= cantidad; indice++) {
+for (let indice = 0; indice < cantidad; indice++) {
   personasEnDisco[indice] = Aleatorio(19, 40);
   console.log(personasEnDisco[indice]);
 }
-console.log(`En la disco hay ${indice - 1} personas`);
+console.log(`En la disco hay ${personasEnDisco.length} personas`);
```

Comment:
- No usamos el indice, que solo va existir dentro del for, porque lo sacamos de las variables globales.
- Simplemente calculamos el largo de arreglo para saber la cantidad de personas.